File: code/1019/1019.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const n = parseInt(lines[0]);

const hours = parseInt(n / 3600);
const minutes = parseInt((n - (hours * 3600)) / 60);
const seconds = parseInt(n - (hours * 3600) - (minutes * 60));

console.log(`${hours}:${minutes}:${seconds}`);

File: code/1019/1019.rb
n = gets.to_i

hours = (n / 3600).to_i
minutes = ((n - (hours * 3600)) / 60).to_i
seconds = (n - (hours * 3600) - (minutes * 60)).to_i

puts format('%d:%d:%d', hours, minutes, seconds)
